### ISSUE TITLE
Server details not updating in server list after upgrading to 469a

### ISSUE THREAD

**Issue by dizzy-egg** [OldUnreal/UnrealTournamentPatches#109]
After upgrading my Linux-based UT server to 469a, the server no longer updates properly in the public server list, or when pinged.

The map remains as the server default map even after it has changed, and the player count always remains at zero even as players join the server.

I haven't changed any settings; only upgraded to 469a.

My master servers are as below:

```
ServerActors=IpDrv.UdpBeacon
ServerActors=IpServer.UdpServerQuery
ServerActors=IpServer.UdpServerUplink MasterServerAddress=master.mplayer.com MasterServerPort=27900
ServerActors=IpServer.UdpServerUplink MasterServerAddress=unreal.epicgames.com MasterServerPort=27900
ServerActors=IpServer.UdpServerUplink MasterServerAddress=utmaster.epicgames.com MasterServerPort=27900
```

**Comment by stijn-volckaert** [comment]
Could you check if this still happens when all players have disconnected?

**Comment by dizzy-egg** [comment]
@stijn-volckaert It does, yes.

**Comment by stijn-volckaert** [comment]
Can you try this version of IpServer.u on your server please? 
[IpServer.zip](https://github.com/OldUnreal/UnrealTournamentPatches/files/5391476/IpServer.zip)

**Comment by dizzy-egg** [comment]
@stijn-volckaert Unfortunately it doesn't seem to have helped. I also updated my master servers on the recommendation of some UT99 server admins, which also hasn't helped.

The server is definitely performing an uplink as I see this in the server log when changing maps:

```
Resolved utmaster.epicgames.com (199.255.40.174)
UdpServerUplink: Master Server is utmaster.epicgames.com:27900
UdpServerUplink: Port 7787 successfully bound.
Resolved master.telefragged.com (108.168.208.29)
UdpServerUplink: Master Server is master.telefragged.com:27500
UdpServerUplink: Port 7787 successfully bound.
Resolved master.333networks.com (84.83.176.234)
UdpServerUplink: Master Server is master.333networks.com:27900
UdpServerUplink: Port 7787 successfully bound.
Resolved gsm.qtracker.com (65.112.87.186)
UdpServerUplink: Master Server is gsm.qtracker.com:27900
UdpServerUplink: Port 7787 successfully bound.
Resolved master.newbiesplayground.net (199.247.4.173)
UdpServerUplink: Master Server is master.newbiesplayground.net:27900
UdpServerUplink: Port 7787 successfully bound.
Resolved master.errorist.eu (3.127.82.233)
UdpServerUplink: Master Server is master.errorist.eu:27900
UdpServerUplink: Port 7787 successfully bound.
```

**Comment by dizzy-egg** [comment]
Oh dear. Reading my own log that I just pasted above I noticed the port numbers being bound were wrong. 

It looks like several instances of `ucc-bin` had been launched and were running simultaneously on the server. I guess only the first one registered the correct port (`7778`) and the rest were forced to register consecutively higher ports. 

I killed all the processes and restarted the server and it seems to be working now. 

This **probably** wasn't related to the patch, although I haven't experienced the issue before now. 

Either way, thank you for your help and sorry if your time's been wasted.
